<?xml version='1.0' encoding='utf-8'?>
<ui version="4.0">
  <class>render_submit</class>
  <widget class="QDialog" name="render_submit">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>700</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>render_submit</string>
    </property>
    <widget class="QLabel" name="headerLabel">
      <property name="text">
        <string>렌더 제출</string>
      </property>
      <property name="geometry">
        <rect>
          <x>250</x>
          <y>10</y>
          <width>100</width>
          <height>30</height>
        </rect>
      </property>
    </widget>
    <widget class="QLabel" name="jobNameLabel">
      <property name="text">
        <string>작업 이름:</string>
      </property>
      <property name="geometry">
        <rect>
          <x>30</x>
          <y>60</y>
          <width>100</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QLineEdit" name="jobNameInput">
      <property name="geometry">
        <rect>
          <x>140</x>
          <y>60</y>
          <width>430</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QLabel" name="sceneLabel">
      <property name="text">
        <string>씬 파일:</string>
      </property>
      <property name="geometry">
        <rect>
          <x>30</x>
          <y>100</y>
          <width>100</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QLineEdit" name="sceneInput">
      <property name="geometry">
        <rect>
          <x>140</x>
          <y>100</y>
          <width>430</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QLabel" name="frameRangeLabel">
      <property name="text">
        <string>프레임 범위:</string>
      </property>
      <property name="geometry">
        <rect>
          <x>30</x>
          <y>140</y>
          <width>100</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QLineEdit" name="frameRangeInput">
      <property name="geometry">
        <rect>
          <x>140</x>
          <y>140</y>
          <width>430</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QLabel" name="outputLabel">
      <property name="text">
        <string>출력 경로:</string>
      </property>
      <property name="geometry">
        <rect>
          <x>30</x>
          <y>180</y>
          <width>100</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QLineEdit" name="outputInput">
      <property name="geometry">
        <rect>
          <x>140</x>
          <y>180</y>
          <width>430</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QLabel" name="renderSettingsLabel">
      <property name="text">
        <string>렌더 설정</string>
      </property>
      <property name="geometry">
        <rect>
          <x>30</x>
          <y>280</y>
          <width>200</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QComboBox" name="rendererCombo">
      <property name="geometry">
        <rect>
          <x>30</x>
          <y>310</y>
          <width>200</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QLabel" name="qualityLabel">
      <property name="text">
        <string>품질:</string>
      </property>
      <property name="geometry">
        <rect>
          <x>250</x>
          <y>310</y>
          <width>50</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QSlider" name="qualitySlider">
      <property name="geometry">
        <rect>
          <x>310</x>
          <y>310</y>
          <width>200</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QCheckBox" name="motionBlurCheck">
      <property name="text">
        <string>모션 블러</string>
      </property>
      <property name="geometry">
        <rect>
          <x>30</x>
          <y>350</y>
          <width>150</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QCheckBox" name="dofCheck">
      <property name="text">
        <string>피사계 심도</string>
      </property>
      <property name="geometry">
        <rect>
          <x>30</x>
          <y>380</y>
          <width>150</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QCheckBox" name="shadowsCheck">
      <property name="text">
        <string>그림자</string>
      </property>
      <property name="geometry">
        <rect>
          <x>30</x>
          <y>410</y>
          <width>150</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QCheckBox" name="reflectionsCheck">
      <property name="text">
        <string>반사</string>
      </property>
      <property name="geometry">
        <rect>
          <x>30</x>
          <y>440</y>
          <width>150</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QLabel" name="priorityLabel">
      <property name="text">
        <string>우선순위:</string>
      </property>
      <property name="geometry">
        <rect>
          <x>30</x>
          <y>500</y>
          <width>80</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QSpinBox" name="prioritySpin">
      <property name="geometry">
        <rect>
          <x>120</x>
          <y>500</y>
          <width>100</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QLabel" name="threadsLabel">
      <property name="text">
        <string>스레드 수:</string>
      </property>
      <property name="geometry">
        <rect>
          <x>250</x>
          <y>500</y>
          <width>80</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QSpinBox" name="threadsSpin">
      <property name="geometry">
        <rect>
          <x>340</x>
          <y>500</y>
          <width>100</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QCheckBox" name="emailCheck">
      <property name="text">
        <string>완료 시 이메일 알림</string>
      </property>
      <property name="geometry">
        <rect>
          <x>30</x>
          <y>550</y>
          <width>200</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QLineEdit" name="emailInput">
      <property name="geometry">
        <rect>
          <x>30</x>
          <y>580</y>
          <width>540</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QPushButton" name="submitButton">
      <property name="text">
        <string>제출</string>
      </property>
      <property name="geometry">
        <rect>
          <x>270</x>
          <y>630</y>
          <width>100</width>
          <height>40</height>
        </rect>
      </property>
    </widget>
    <widget class="QPushButton" name="validateButton">
      <property name="text">
        <string>검증</string>
      </property>
      <property name="geometry">
        <rect>
          <x>380</x>
          <y>630</y>
          <width>100</width>
          <height>40</height>
        </rect>
      </property>
    </widget>
    <widget class="QPushButton" name="cancelButton">
      <property name="text">
        <string>취소</string>
      </property>
      <property name="geometry">
        <rect>
          <x>490</x>
          <y>630</y>
          <width>80</width>
          <height>40</height>
        </rect>
      </property>
    </widget>
  </widget>
</ui>
